packages/terra-card/src/Card.jsx

```diff
@@ -0,0 +1,47 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import classNames from 'classnames/bind';
+import 'terra-base/lib/baseStyles';
+import styles from './Card.scss';
+
+const cx = classNames.bind(styles);
+
+const propTypes = {
+  /**
+   * Provides themeable 10px on top and bottom
+   */
+  padVertical: PropTypes.bool,
+  /**
+   * Provides themeable 10px on left and right
+   */
+  padHorizontal: PropTypes.bool,
+  /**
+   * Child Nodes
+   */
+  children: PropTypes.node,
+};
+
+const defaultProps = {
+
+};
```

Comment:
Do we plan to remove the defaultProps since they turn out to be undefined and end up not setting the className, or do we plan to explicitly set them to false?